### DISCUSSION TITLE
[Feature] View log(s) from web GUI

### DISCUSSION THREAD

**Discussion by blublub** [immich-app/immich#10937] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi,
As the title says: Viewing logs from within the web-GUI - I believe this would be a handy feature for non-tech-savy users to provide information on problems and bugs

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3484

**Reply by blublub** [reply]
> #3484

thx - it seems I have to search better...
